src/prognostic_equations/edmfx_sgs_flux.jl

```diff
@@ -97,6 +200,12 @@ function edmfx_sgs_mass_flux_tendency!(
     (; edmfx_sgsflux_upwinding) = p.atmos.numerics
     (; ᶠu³, ᶜh_tot, ᶜspecific) = p.precomputed
     (; ᶜρaʲs, ᶜρʲs, ᶠu³ʲs, ᶜKʲs, ᶜmseʲs, ᶜq_totʲs) = p.precomputed
+    if (
```

Comment:
No need to change this function as this is for diagnostic EDMF.

Reply:
I got ahead of myself. But some day...